packages/kbn-alerts-as-data-utils/src/field_maps/legacy_experimental_field_map.ts

```diff
@@ -37,12 +37,12 @@ export const legacyExperimentalFieldMap = {
   },
   [ALERT_GROUP_FIELD]: {
     type: 'keyword',
-    array: false,
+    array: true,
```

Comment:
@ymao1 Is it safe to fix this field mapping? array: false was wrong and it should have been true.